macros.in

```diff
@@ -335,16 +335,16 @@ package or when debugging this package.\
 
 #	A colon separated list of desired locales to be installed;
 #	"all" means install all locale specific files.
-#	
```

Comment:
Why give someone a reason to reject this just because it includes unrelated whitespace changes?

Reply:
Indeed...